### ISSUE TITLE
In application form, use `<h1>`s for program names

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/civiform#1539]
Screen readers benefit from having exactly one `<h1>` on each page. The application pages (where applicants can fill out program forms) lack `<h1>`s, so we should probably upgrade the program name (which is currently a blue `<h2>`) to an `<h1>`.

In the image below, that would be the "program for question lifecycle" heading.

![image](https://user-images.githubusercontent.com/77128518/124012881-a2df9d00-d9af-11eb-9fa8-580f6ed77b4e.png)

**Comment by neel-a-mehta** [comment]
Aha! I did a closer look and it appears that there is an `sr-only` `<h1>` tag hiding on the page. I think that's good then
